server/routes/api/helpscout.ts

```diff
@@ -0,0 +1,126 @@
+import * as express from "express";
+import {isHelpScoutIntegrationEnabled, validateHelpScoutSignature} from "../../middleware";
+import {IFormItem, IUser, User} from "../../schema";
+import bodyParser = require("body-parser");
+import * as moment from "moment-timezone";
+
+export const helpScoutRoutes = express.Router({"mergeParams": true});
+
+export type RequestWithRawBody = express.Request & { rawBody: string };
+
+helpScoutRoutes.route("/userInfo").post(
+	isHelpScoutIntegrationEnabled,
+	bodyParser.json({
+		verify: (req: RequestWithRawBody, res, buffer, encoding) => {
+			if (buffer && buffer.length) {
+				req.rawBody = buffer.toString(encoding || 'utf-8');
+			}
+		}
+	}),
+	validateHelpScoutSignature,
```

Comment:
Maybe move the middleware up a level to the routes file, so it's included for all routes  under `helpScoutRoutes` if any are added in the future.

Reply:
Done!